app/src/test/java/com/codesoom/assignment/controller/ProductUpdateControllerTest.java

```diff
@@ -0,0 +1,90 @@
+package com.codesoom.assignment.controller;
+
+import com.codesoom.assignment.application.ProductNotFoundException;
+import com.codesoom.assignment.application.ProductUpdateServiceImpl;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductDto;
+import com.codesoom.assignment.domain.ProductRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+
+@ActiveProfiles("test")
+@SpringBootTest
+public class ProductUpdateControllerTest {
+
+    private ProductUpdateController controller;
+
+    @Autowired
+    private ProductUpdateServiceImpl service;
+
+    @Autowired
+    private ProductRepository repository;
+
+    @BeforeEach
+    void setup() {
+        this.controller = new ProductUpdateController(service);
+        repository.deleteAll();
+    }
+
+    @DisplayName("update 메서드는")
+    @Nested
+    class Describe_update {
+
+        private final ProductDto productDto
+                = new ProductDto("소쩍새", "유령회사", BigDecimal.valueOf(3000), "");
```

Comment:
업데이트 요청에 사용할 용도라고 명시해주면 더 읽기 좋겠죠.
```suggestion
        private final ProductDto productToUpdate
                = new ProductDto("소쩍새", "유령회사", BigDecimal.valueOf(3000), "");
```